### PR TITLE
issue-9; progress on creating combat challenges

### PR DESCRIPTION
**Pull request by glynyon-5020**
Not ready yet. Just making sure everything works.

### CODE REVIEW

**Comment by sigrunixia** [comment]
Moved this over to main as we actually dont need releases anymore. :)

**Comment by glynyon-5020** [comment]
Alrighty. Ready for review! Roast me, please. I'm new to trying to transpose books and tagging and collaborating on open source projects. :)

File: Compendium/WWN/Creating Adventures/Creating Combat Challenges.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+aliases: [Creating Combat Challenges]
+
+page: 230
+
+tags: WWN/Creation
+
+title: Creating Combat Challenges
+
+---
```
Comment (sigrunixia) [inline Compendium/WWN/Creating Adventures/Creating Combat Challenges.md:17]: Frontmatter duplication here :)

File: Compendium/WWN/Creating-Adventures/Creating-Combat-Challenges.md [hunk 2]
```diff
@@ -1,6 +1,195 @@
...
+    - The target may have employed or forced others into accomplishing their aims
+    - Populating a [[Deep]] or some other highly-populated location of monsters, the creatures likely belong in that environment, or have adapted to live in the area
+3. Avoid thinking about "balancing" combat challenges
+    - Don't adjust enemy numbers to make the encounter easier or harder based on the power level of the party; rather, tell the narrative in a logical and coherent way such that the PCs can engage with it
+    - i.e. The King's Guard are ferocious, the Anak tribe has scores of combat-capable warriors, the evil sorcerer is surrounding by their inhuman minions, and the most infamously dangerous swordsman in the country regardless of whether your party is level one or level ten
+4. Think about the context of your adventure's purpose
+   - A [[Deep]] for novice heroes should contain winnable combat challenges
+   - There should be a logical reason for why the [[Deep]] is filled with PC-slaughtering hell-beasts
+   - Have trivial and grand foes
+   - Refer to [[page 295]] for aid in assessing a combat challenge's likelihood of killing PCs
```
Comment (glynyon-5020) [inline Compendium/WWN/Creating-Adventures/Creating-Combat-Challenges.md:51]: is there a way to query a file with the page 295 metadata?

File: Compendium/WWN/Creating-Adventures/Creating-Combat-Challenges.md [hunk 3]
```diff
@@ -1,6 +1,195 @@
...
+2. Think about any surrounding threats, associates, or minions that it might have
+    - Merchants have guards, cult leaders have underling fanatics, and Anak warriors often roam with their warbands
+    - Allied groups may have individuals in the area
+    - The target may have employed or forced others into accomplishing their aims
+    - Populating a [[Deep]] or some other highly-populated location of monsters, the creatures likely belong in that environment, or have adapted to live in the area
+3. Avoid thinking about "balancing" combat challenges
+    - Don't adjust enemy numbers to make the encounter easier or harder based on the power level of the party; rather, tell the narrative in a logical and coherent way such that the PCs can engage with it
+    - i.e. The King's Guard are ferocious, the Anak tribe has scores of combat-capable warriors, the evil sorcerer is surrounding by their inhuman minions, and the most infamously dangerous swordsman in the country regardless of whether your party is level one or level ten
+4. Think about the context of your adventure's purpose
+   - A [[Deep]] for novice heroes should contain winnable combat challenges
```
Comment (glynyon-5020) [inline Compendium/WWN/Creating-Adventures/Creating-Combat-Challenges.md:48]: figured maybe a definition note for Deep was warranted

File: Compendium/WWN/Creating-Adventures/Creating-Combat-Challenges.md (hunk 2 above)
Comment (sigrunixia) [reply Compendium/WWN/Creating-Adventures/Creating-Combat-Challenges.md:51]: Not off the top of my head. What I plan to do is create a direct link to the page in question but leave the alias as [[path-to-page|page 295]].

File: Compendium/WWN/Creating-Adventures/Creating-Combat-Challenges.md (hunk 3 above)
Comment (sigrunixia) [reply Compendium/WWN/Creating-Adventures/Creating-Combat-Challenges.md:48]: That can be done. I'll create a Glossary folder for such things.

**Review by sigrunixia — APPROVED** [review]

**Pull request merged**